Bottom: ba3eca830dd99d2bfcdb5628551aeda976f32bc2
Top:    7339c5d937e457ecb893618d66bd6bcd91b15ebe
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-11 16:51:50 +0000

Reduce the redundant work done in node_label


---

diff --git a/src/polygon1.c b/src/polygon1.c
index f9fa617..f472fdc 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1100,14 +1100,17 @@ label_contour (PLINE * a)
           continue;
         }
 
+#if 0
       if (NODE_LABEL (cur) != UNKNWN)
         {
+          fprintf (stderr, "Not expected...\n");
           /* This might get hit if the last cross vertex labelling
            * pass was a bit keen, and labeled past its own edges?
            */
           label = NODE_LABEL (cur);
           continue;
         }
+#endif
 
       if (label == INSIDE || label == OUTSIDE)
         {
